src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,89 @@
+@using OrchardCore.Forms.ViewModels
+@model SelectPartEditViewModel
+
+<script asp-src="~/OrchardCore.Forms/Scripts/selectOptionsEditor.min.js" debug-src="~/OrchardCore.Forms/Scripts/selectOptionsEditor.js" asp-name="selectOptionsEditor" at="Foot" depends-on="vuejs, vuedraggable"></script>
+<style asp-src="~/OrchardCore.Forms/Styles/selectOptionsEditor.min.css" debug-src="~/OrchardCore.Forms/Styles/selectOptionsEditor.css"></style>
+
+<script at="Foot">
+    initializeSelectOptionsEditor(document.getElementById('@Html.IdFor(m => m)'), @Html.Raw(Model.Options), '@Model.DefaultValue', document.getElementsByClassName('@Html.IdFor(m => m)-ModalBody'));
+</script>
+
+<script type="text/x-template" id="select-options-table">
+    <table class="table table-bordered table-sm select-options-table">
+        <thead class="thead-light">
+            <tr>
+                <th scope="col">@T["Option Text"]</th>
+                <th scope="col">@T["Value"]</th>
+                <th scope="col" colspan="3">@T["Default?"]</th>
+            </tr>
+        </thead>
+        <draggable v-model="data.options" :tag="'tbody'">
+            <tr v-for="(option, index) in data.options" :key="index">
+                <td>
+                    <input type="text" class="form-control" v-model="option.text" placeholder="Enter the text" />
+                </td>
+                <td>
+                    <input type="text" class="form-control courier" v-model="option.value" placeholder="Enter a value" />
```

Comment:
@T["Enter a value"]

Reply:
done.

@agriffard we should probably update the `TextFieldPredefinedListEditorSettings.Edit` as that was copied straight over, and they're missing there too